optuna/visualization/_parallel_coordinate.py

```diff
@@ -37,10 +37,10 @@ class _DimensionInfo(NamedTuple):
     label: str
     values: Tuple[float, ...]
     range: Tuple[float, float]
-    is_log: bool = False
-    is_cat: bool = False
```

Comment:
Since these values are not mutable, so we can use them as default values. But Let's address them as a follow-up.